fonduer/matchers.py

```diff
@@ -228,11 +228,18 @@ def init(self):
         self.attrib = self.opts.get('attrib', WORDS)
         self.sep = self.opts.get('sep', " ")
 
+        # Extending the RegexMatch to handle search(instead of only match) 
+        # and adding a toggle for full span match.
+        # Default values are set to False and True for search flag and full 
```

Comment:
Please get rid of these trailing whitespaces.

Make sure the code still passes our code style check: `make check`

Reply:
Done